agent/php_execute.c

```diff
@@ -93,6 +93,12 @@
  * conditions wherever possible.
  */
 
+static inline nruserfn_t* nr_php_get_wraprec_from_op_array_extension(const char* fn, zend_function* zf) {
+  nruserfn_t* wraprec = (nruserfn_t*)ZEND_OP_ARRAY_EXTENSION(&zf->op_array, NR_PHP_PROCESS_GLOBALS(op_array_extension_handle));
+  nrl_verbosedebug(NRL_AGENT, "%s from %s, op_array_extension=%p, wraprec=%p, wraprec->pid=%d", __func__, fn, ZEND_OP_ARRAY_EXTENSION(&zf->op_array, NR_PHP_PROCESS_GLOBALS(op_array_extension_handle)), wraprec, wraprec->pid);
```

Comment:
This will segfault if `fn` is NULL.

Needs a check if `zf` is NULL before dereferencing it.

Reply:
`nr_php_get_wraprec_from_op_array_extension` has been removed.